specification/components/schemas/AdviceAndGuidanceCommunicationBundle.yaml

```diff
@@ -0,0 +1,150 @@
+title: Bundle resource containg advice and guidance conversation
+type: object
+required:
+  - id
+  - meta
+  - resourceType
+  - type
+  - total
+  - link
+  - entry
+properties:
+  id:
+    type: string
+    example: '08acc887-6a7b-4b38-8f50-022e41319efe'
+  meta:
+    type: object
+    required: 
+      - lastUpdated
+      - profile
+    properties:
+      lastUpdated:
+        type: string
+        format: date-time
+        example: '2021-04-20T12:17:35.128Z'
+      profile:
+        type: array
+        items:
+          type: string
+          enum:
+            - 'http://hl7.org/fhir/STU3/StructureDefinition/Bundle'
+          example: 'http://hl7.org/fhir/STU3/StructureDefinition/Bundle'
+  resourceType:
+    type: string
+    enum:
+      - Bundle
+    example: 'Bundle'
+  type:
+    type: string
+    enum:
+      - searchset
+    example: 'searchset'
+  total:
+    type: integer
+    minimum: 0
+    format: int32
+    example: 3
+  link:
+    type: array
+    minItems: 1
+    maxItems: 3
```

Comment:
I don't think we support paging for the get communication entries, so I guess the maximum here would be 1?